binr/rax2/rax2.c

```diff
@@ -480,7 +482,26 @@ static int rax(char *str, int len, int last) {
 			free (str);
 		}
 		return false;
-	}
+	} else if (flags & (1 << 21)) { // -i
+                printf("unsigned char buf[] = {");
+                /* resonable amount of bytes per line */
+                int byte_per_col = 12;
+		for (i = 0; i < len-1; i++) {
+                        /* wrapping every N bytes */
+                        if (i % byte_per_col == 0) {
+                                printf("\n  ");
+                        }
+			printf ("0x%02x, ", (ut8) str[i]);
+		}
+                /* some care for the last element */
+                if (i % byte_per_col == 0) {
+                        printf("\n  ");
+                }
+                printf ("0x%02x\n", (ut8) str[len-1]);
```

Comment:
underflow if len == 0

Reply:
The -I case is unreachable with len == 0, should I still check for it?